packages/mock-addon-docs/.storybook/main.ts

```diff
@@ -22,6 +18,7 @@ const config = {
                 },
             },
         },
+        import.meta.resolve('./local-preset.ts'),
```

Comment:
New way of loading addons locally in SB10: you must import them. I've realigned local-preset on what we do on addon-kit.